src/sharing/components/ShareByEmail.jsx

```diff
@@ -161,14 +161,18 @@ class ShareByEmail extends Component {
             count: recipients.length
           })
         }
+        this.reset()
       })
       .catch(err => {
         Alerter.error('Error.generic')
-        throw err
-      })
-      .finally(() => {
         this.reset()
+        throw err
       })
+    // TODO: We'd like to use finally here but unfortunately it doesn't work on FF
+    // even with the `shippedProposals` option to babel-preset-env
+    // .finally(() => {
+    //   this.reset()
+    // })
```

Comment:
So we should remove https://github.com/cozy/cozy-drive/pull/653/commits/a43e28e09d0adfafd78f4b752b07355d5078e6ef#diff-e56633f72ecc521128b3db6586074d2cR9.